sdks/java/io/elasticsearch/pom.xml

```diff
@@ -109,7 +109,6 @@
     <dependency>
       <groupId>org.hamcrest</groupId>
       <artifactId>hamcrest-core</artifactId>
-      <version>1.3</version>
       <scope>test</scope>
```

Comment:
I think we can remove scope scope here, it should inherit from the dependency management in the parent pom.